sale_order_product_assortment/models/sale_order.py

```diff
@@ -28,17 +29,52 @@ def _compute_product_assortment_ids(self):
         product_domain = []
         if self[partner_field]:
             filters_partner_domain = self.env["ir.filters"].search(
-                [("is_assortment", "=", True)]
+                [
+                    ("is_assortment", "=", True),
```

Comment:
Don't introduce a comma here for avoiding this unneeded diff.

Reply:
Fixed, thanks